code/datums/martial.dm

```diff
@@ -556,11 +558,15 @@
 							"<span class='userdanger'>[A] attempted to disarm [D]!</span>")
 		playsound(D, 'sound/weapons/punchmiss.ogg', 25, 1, -1)
 	add_logs(A, D, "disarmed with CQC")
-	if(restrained)
-		D.visible_message("<span class='danger'>[A] puts [D] into a chokehold!</span>", \
-							"<span class='userdanger'>[A] puts you into a chokehold!</span>")
-		D.SetSleeping(20)
-		restrained = 0
+	if(restraining)
+		if(D = current_target)
```

Comment:
= means you are setting D to current_target. I haven't read the rest of the code so take this with a grain of salt but usually if()s use ==, since that's for checking if D is already set to current_target.